File: 14/14.py
n=int(input("Enter a number:"))
for rows in range(1,n+1,1):
    for cols in range(n,rows,-1):
        print(" ",end="")
    print("*",end="")
    for cols in range(1,(rows-1)*2,1):
        print(" ",end="")
    if rows==1:
        print("\n",end="")
    else:
        print("*\n",end="")
    
for rows in range(n-1,0,-1):
    for cols in range(n,rows,-1):
        print(" ",end="")
    print("*",end="")
    for cols in range(1,(rows-1)*2,1):
        print(" ",end="")
    if rows == 1:
        print("\n",end="")
    else:
        print("*\n",end="")

File: 14/14.c
#include <stdio.h>

int main() {
    int n,cols,rows;
    printf("Enter a number: ");
    scanf("%d",&n);
    for(rows=1;rows<=n;rows++){
        for(cols=n;cols>rows;cols--){
            printf(" ");
        }
        printf("*");
        for(cols=1;cols< (rows-1)*2;cols++){
            printf(" ");
        }
        if(rows == 1){
            printf("\n");
        }
        else{
            printf("*\n");
        }
    }
    
    for(rows = n-1;rows>=1;rows--){
        for(cols = n;cols>rows;cols--){
            printf(" ");
        }
        printf("*");
        for(cols = 1;cols<(rows-1)*2;cols++){
            printf(" ");
        }
        if(rows==1){
            printf("\n");
        }
        else{
            printf("*\n");
        }
    }
    return 0;
}
